crux-docker/README.md

```diff
@@ -0,0 +1,70 @@
+See basic (non-configured) setup instructions on [docker hub](https://hub.docker.com/r/juxt/crux-http). 
+
+To create a custom configured node, download the
+contents of this folder and read the instructions below.
+
+--- 
+## Getting Started
+
+The only requirement for setting up this node is **docker**, so ensure that is installed. Firstly, if you have not already done so, 
+download the most recent image of the http node from docker hub:
+
+```bash
+docker pull juxt/crux-http
+```
+
+To run the basic instance of the node (which runs using the **memdb**), run the script within `./bin/dev.sh`. This image makes use of 
```

Comment:
can we put the `docker run` command in here, so that users don't need to clone the repo for the most basic use case? (I don't think you even need to `docker pull` in that case, `docker run` might do it for you)

Reply:
ah, I see - those instructions are on the Docker Hub page, guessing on the assumption that people coming here won't mind cloning the repo.

Reply:
Yeah, exactly ^. The 'basic' version is on the docker hub.